doc-content/enterprise-only/getting-started-kogito/con-migrate-dmn-to-kogito-overview.adoc

```diff
@@ -0,0 +1,65 @@
+[id="con-migrate-dmn-to-kogito-overview_{context}"]
+= Migration of a DMN service to {KOGITO} microservice
+
+[role="_abstract"]
+You can migrate decision services that are implemented using DMN models to {KOGITO} microservices easily by moving the DMN resources from KJAR artifacts to the respective {KOGITO} archetype. In the {KOGITO} microservices, some of the KIE v7 features are no longer required.
+
+
+[id="ref-migrate-dmn-to-kogito-considerations_{context}"]
+== Major changes and migration considerations
```

Comment:
_Suggest capitalizing all sentences starts, as in "Stored in ...", "Copy ...", "Managed using ..." instead of "stored in ...", etc. Not sure of the rationale, and more correct/common to cap all._
I would keep it lower case, as the sentences in the table is a continuation to the header, for example:

In {KIE_SERVER} API, stored in `src/main/resources` of KJAR. 

_Also per IBM, suggest end all with a period. In tables and lists, if one of the close components is a complete sentence requiring a period, then all other nearby entries should likewise carry a period for consistency, even when not complete sentences. So would be "Stored in src/... of KJAR." and "Copy as is to src/main/resources." etc._

agree, adding periods to all sentences.